src/context/reducers/network.ts

```diff
@@ -1,13 +1,8 @@
 import buildings from "../../buildings";
 import { State } from "../store";
 
-export type GameMode =
-  | "Ethereum"
-  | "Polygon"
-  | "Goerli"
-  | "Christmas"
-  | "Sepolia"
-  | "LunarNewYear";
+export type GameMode = "Ethereum" | "Polygon" | "Sepolia";
+export type LeaderboardGameMode = GameMode | "Christmas" | "LunarNewYear";
```

Comment:
is `| "Christmas" | "LunarNewYear"` necessary here? Since it's being removed everywhere else
```suggestion
export type LeaderboardGameMode = GameMode;
```